libs/utils/utils.js

```diff
@@ -442,3 +443,24 @@ export function getBlockClasses(className) {
   const variants = blockWithVariants.map((v) => trimDashes(v));
   return { name, variants };
 }
+
+export async function fetchPlaceholders() {
+  if (!window.placeholders) {
+    const resp = await fetch(`${window.location.origin}/placeholders.json`);
+    const json = await resp.json();
+    window.placeholders = {};
+    json.data.forEach((placeholder) => {
+      window.placeholders[placeholder.Key] = placeholder.Text;
+    });
+  }
+  return window.placeholders;
+}
+
+export function debug(message) {
```

Comment:
instead of this `debug` I think we should use lana.  It will automatically log to console for non-prod, and if the footer is failing in prod we'd want to know (it'll show up in the splunk logs)

Reply:
I'll resolve this in MWPW-116224